src/main/java/com/cleanroommc/multiblocked/common/capability/EnergyGTCECapability.java

```diff
@@ -64,7 +68,10 @@ public Long copyInnerByModifier(Long content, ContentModifier modifier) {
 
     @Override
     public EnergyCapabilityProxy createProxy(@Nonnull IO io, @Nonnull TileEntity tileEntity) {
-        return new EnergyCapabilityProxy(tileEntity);
+        EnergyCapabilityProxy energyCapabilityProxy = new EnergyCapabilityProxy(tileEntity);
+        energyCapabilityProxy.facing = getFrontFacing(GregtechCapabilities.CAPABILITY_ENERGY_CONTAINER, tileEntity);
+        System.out.println("TTT : setting facing to " + energyCapabilityProxy.facing);
```

Comment:
Remove this println debug

Reply:
Ugh sorry, thought I'd removed that. Just ran a quick last minute check on it.